app/src/main/java/com/hedvig/app/feature/loggedin/ui/LoggedInActivity.kt

```diff
@@ -95,7 +95,7 @@ class LoggedInActivity : BaseActivity(R.layout.activity_logged_in) {
                 if (isDarkTheme) {
                     gradient.drawable?.alpha = boundedLerp(127, 255, positionInSpan / 5)
                 } else {
-                    gradient.drawable?.alpha = boundedLerp(255, 0, positionInSpan / 5)
+                    gradient.drawable?.alpha = boundedLerp(255 * 0.6f, 0f, positionInSpan / 5).toInt()
```

Comment:
Font for light mode should be at max 60% alpha according to design. This changes this.